tide_landing_page.install

```diff
@@ -1655,3 +1655,112 @@ function tide_landing_page_update_8036() {
   $config_read = _tide_read_config($json_field, $config_location, TRUE);
   $config_storage->write($json_field, $config_read);
 }
+
+/**
+ * Replace featured_news with navigation card.
+ */
+function tide_landing_page_update_8037() {
+  $moduleHandler = \Drupal::service('module_handler');
+  if (!$moduleHandler->moduleExists('tide_news')) {
+    return;
+  }
+  $paragraph_query = \Drupal::entityTypeManager()
+    ->getStorage('paragraph')
+    ->getQuery();
+  $paragraph_query->condition('parent_type', 'node')
+    ->condition('parent_field_name', 'field_landing_page_component')
+    ->condition('type', 'featured_news');
+  $paragraph_ids = $paragraph_query->execute();
+  if ($paragraph_ids) {
+    foreach ($paragraph_ids as $paragraph_id) {
+      $paragraph = Paragraph::load($paragraph_id);
+      $nodes = \Drupal::entityTypeManager()
+        ->getStorage('node')
+        ->loadByProperties(['field_landing_page_component' => $paragraph->id()]);
```

Comment:
Please check $paragraph instanceof.

Reply:
It doesn't need. why should I check?

Reply:
Because paragraph load can return null.

Reply:
in this case, it will never return null, as the ids came from entity query. if it does return a null, it will be an exception, drupalcore will handle this.